Sources/ParseSwift/Storage/ParseFileManager.swift

```diff
@@ -83,11 +83,19 @@ internal struct ParseFileManager {
     }
 
     init?() {
+        #if os(Linux)
+        guard let applicationId = ParseConfiguration.applicationId else {
+            return nil
+        }
+        applicationIdentifier = "com.github.parse-community.parse-swift.\(applicationId)"
```

Comment:
I think we can use the same as the iOS, macOS, etc., but can leave the applicationId appended as you have it. So:

`com.parse.ParseSwift.\(applicationId)`